tests/components/esphome/test_text.py

```diff
@@ -0,0 +1,121 @@
+"""Test ESPHome texts."""
+
+import math
+from unittest.mock import call
+
```

Comment:
With tests ❤️!

Reply:
Cant ruin all the hard work you did to get the coverage up =)